src/te_checks.c

```diff
@@ -663,6 +701,77 @@ struct check_result *check_unknown_interface_call(__attribute__((unused)) const
 				 if_name);
 }
 
+static bool is_bare_parameter(const char *name)
```

Comment:
Is "bare parameter" a common term that I'm just not aware of?  I had to read the code pretty closely to figure out what this was actually checking.  So unless we expect policy developers to know/use that term and I'm just the one out of the loop here, can we add a comment defining "bare parameter" above this function?

Reply:
Added a comment to the function, but I am also happy to rename it; suggestions?

Reply:
If there's a commonly accepted term for the distinction you're making here, I'm not sure what it is.  I do like "simple" as used in your comment better than "bare" I think.  I don't know that I have a better suggestion.  I think the example in the comment clarifies the intent quite nicely.